promql/functions.go

```diff
@@ -603,6 +604,40 @@ func funcChanges(ev *evaluator, args Expressions) Value {
 	return out
 }
 
+// === label_replace(vector ExprVector, dst_label, replacement, src_labelname, regex ExprString) Vector ===
+func funcLabelReplace(ev *evaluator, args Expressions) Value {
+	var (
+		vector   = ev.evalVector(args[0])
+		dst      = clientmodel.LabelName(ev.evalString(args[1]).Value)
```

Comment:
It's always a string, but we need to eval it from being a string expression node into its actual string value first via `evalString`.